### ISSUE TITLE
Suggestion: Server setting for max viewdistance

### ISSUE THREAD

**Issue by Dreepa** [StrangeLoopGames/EcoIssues#7014]
Suggestion: 
In Minecraft the server can overrule the client as to how many chunks the client will load around the player.
This helped with performance, as server had to "wake up" less chunks when players were running around the world.

Maybe this might be an idea for ECO as well (if the game works similar, I don't know of course).

**Comment by darkshad9999** [comment]
Not needed the max distance view is determined on what hardware (video card, CPU etc) that player as to what the max view distance will be. The server should never overrule the client. This would be a disaster. Please don't put this in the game devs!

**Comment by Dreepa** [comment]
This suggestion wasn't about gfx power, but data transmission of chunks updated around each player's position.

**Comment by Dreepa** [comment]
Are you insane or something? I made this thread, you post in it (again something totally irrelevant to the topic, again without any technical knowledge, again with an attitude of "OMG THE WORLD IS GOING DOWN IN FLAMES" and again without any choice of words what would enable your discussion partner to take you seriously) and then I respond to you, where after you accuse me of following you?!?

Did it ever occur to you, that you have no idea what you are talking about, and that you still have a lot to learn? Do you ever approach any given topic with the possibility of you being wrong or of you having made an erroneously assumption? No, you don't. You just spout out the first thought that comes to your brain and act like you are the authority of things here. You are the pinnacle of egocentric hubris. 

So before you accuse someone of following you, how about you check who you respond to, you master genius.

**Comment by darkshad9999** [comment]
I am sick of you Dreepa you asshole. Fuck off now!  You are nothing more than a troll and an asshole!

**Comment by darkshad9999** [comment]
You are blocked Dreepa I am sick of your harassment

**Comment by Dreepa** [comment]
What is wrong with this guy?

**Comment by darkshad9999** [comment]
Last2night no it doesn't That is done by the CLIENT not the server the settings already basically exist 
on the client for those kind of settings they do NOT need to be put server side. 

As I said before the option already exists on the client in the form of the max view distance
and the max field of view. 

And Last2Night I can post a response about a suggestion made here if I want piss off. 

This is not needed and I again would ask that the devs not put anything like this 
on the server side of the game again I would not trust any of these people that
are running these servers to actually set the option properly and there 
would be no reason for the server to override the settings.

**Comment by Last2Night** [comment]
@darkshad9999 Are you a part of devs team to make such statement? Perhaps you have access to game's source code that confirms your statement? Or is it just your assumption?

You've already said that you are against this request 3 or 4 times, there is no need to duplicate it in every response you post, I assume devs heard your opinion.

**Comment by darkshad9999** [comment]
Last2Night go away you idiot yet another stalking asshole.. fuck off and stop trolling me! 

Someone please ban this asshole from the github tracker please!  You are being reported.

**Comment by TheGhost10000** [comment]
I would agree that something like this isn't really needed.  The player should be the one setting the option for the chunk size.  And this is done by two options which are already included in the client being the field of view and the view distance option.

The server admin should not need to or be allowed to set the chunk size to be loaded that should 
be determined by the client not the server.

What happens if the server admin sets the chunk size too large and a person with a lower
end graphics card then tries to connect to that server. 

How does the server admin even determine what the correct chunk size  or number of chunks that would be that they should be loading. 

And it isn't going to put any more of a load on the server CPU either and if you have a proper 
server there should be little to know effect. 

I would say over 99% of the people who are currently running servers should not be running
servers in the first place and have no clue what they are doing. I would never trust one of 
the server admins to be able to override the correct chunk size or the number of chunks to load 
and be able to set it properly. 

Again this should be handled by the client NOT the server because the number of chunks or the chunk
size that can be handled is going to depend on the individual hardware.

This isn't Minecraft and an option like this really isn't needed on the server.

**Comment by Last2Night** [comment]
@TheGhost10000 
I am an ECO server owner. I host an ECO server since it was released on Steam. Our server is not rented from third party game servers provider, we run the truly dedicated one. I can easily determine what stresses my server and what is best for the server to run lag free, and control over the amount of load being put on server is the most important thing.

If an Admin runs a server and does not know what he is doing, thats not a part of this story. Dont play on servers that are not properly maintained.

If you want to get rid of bad servers, I suggest you start a suggestion to get rid of third party providers instead of limiting administration abilities.

**Comment by TheGhost10000** [comment]
Then you are not hosting it on a proper server Last2Night and that is the issue.  Running a game server off of an old PC that you have kicking around the house is not a good idea.  I am a server admin and have gone to school to be a Network Admin myself. I know what I am talking about here. 

Get a proper server then because this game is not going to run well off of anything else on a DSL or Cable modem connection. 

Once again hosting is very cheap for an Eco server and there are several out there. 

Once again if you are running this game off of a professional server you would
not see any issues that you are claiming with this game. 

Why would I make a suggestion to get rid of third party providers that is where
you should be hosting the server at a proper hosting company with proper 
servers. 

No one is limiting an administrators ability here you are limited yourself 
by running this server on old outdated equipment. 

Once again this option isn't needed.

**Comment by Last2Night** [comment]
@TheGhost10000 
False claims points out that you have no idea what you are talking about. The community based on our ECO has over 50 players. They wouldnt use our services if we wouldnt know what we are doing. I suspect that you are running your server from your home computer since you mentioned DSL/Cable? If so then we have nothing to discuss here as you have no server administration experience.

**Comment by TheGhost10000** [comment]
There is no false claims being made here... I do know what I am talking about. 

The Eco server from what you said before does NOT sound like it is being run on 
any kind of a professional server and that is why you are having the issues you are. 

And I don't run a public server I just have a test server here for my OWN personal
testing and that is it.  

LOL I have no server administration experience ya right you don't know me 
why don't you stop attacking me and insulting me and go get yourself 
an actual server that is professionally hosted. 

That is why I went to school on two occasions and graduated with 
a Computer information systems diploma and then have since 
gone back to school and did my Microsoft Certified Solutions Expert (MCSE), 
Comp TIA A+ certification, Cisco (CCNA), Unix Administration

I will forget more about computers then you will EVER know. So please 
cease the insults now! 

Any more insults or attacks from you and I will report you.

**Comment by Last2Night** [comment]
@TheGhost10000 
Have a good day. Our discussion is over.

**Comment by TheGhost10000** [comment]
Thought so wouldn't even give me any kind of hardware information on the server that your eco
server is running on.

Devs can you please close this ticket this isn't anything that you need to implement 
and is an issue with the hardware that they are running the Eco server on.

Thank you,

**Comment by BlazeGuy2** [comment]
This issue was moved to StrangeLoopGames/EcoSuggestions#154
